src/apis/AsyncStorage.md

```diff
@@ -4,6 +4,15 @@ title: AsyncStorage
 wip: true
 ---
 
+`AsyncStorage` has been deprecated and will be removed from React Native in the
```

Comment:
Do we want to take this even further and deprecate the `AsyncStorage` as well, by adding a `[@ocaml.deprecated]` in the `AsyncStorage.rei` file? Then users will get the message right in the compiler.

Reply:
I proposed the same in #594 but @MoOx rightly pointed out that React Native has deprecation warnings, so users are presumably aware that they are using a deprecated component or API.

We could have deprecation warnings pointing to replacement bindings. I suppose they are a bit annoying when you cannot migrate to another package, but can be turned in `bsconfig.json`.

Reply:
You have RN deprecating warnings in simulator + in terminal, pretty sure that's more than enough.

Reply:
Already discussed in an other PR, I think RN does what it need to warn user already by its own.